lib/connection-parameters.js

```diff
@@ -50,8 +54,11 @@ var ConnectionParameters = function (config) {
     config = Object.assign({}, config, parse(config.connectionString))
   }
 
+  var dbDefaulting = function () { return this.user || defaults['database'] }
```

Comment:
Is it possible for `this.user` to be falsy?

Reply:
Looking at the implementation, it's set with `val('user', config)` which can be false when (a) there is no default set and (b) the corresponding environment variable is not set.

From the outside, URLs like `postgres:///` don't have a user set, so it stands to reason the implementation has to allow it to be falsey.

Reply:
The default has to be set to something; if it’s `undefined`, the connection is going to fail anyway. I think this can just be `const getDefaultDatabase = () => this.user`.